File: JS-Basics/Conditional Statements/Conditional Statements - Exercise/toyShop.js
function toyShop(input) {
    let puzzle = 2.6;
    let doll = 3;
    let teddyBear = 4.1;
    let minion = 8.2;
    let truckToy = 2;

    let tripPrice = Number(input[0]);
    let puzzlesAmount = Number(input[1]);
    let dollsAmount = Number(input[2]);
    let teddyBearsAmount = Number(input[3]);
    let minionsAmount = Number(input[4]);
    let truckToyAmount = Number(input[5]);
    let totalAmount = puzzlesAmount + dollsAmount + teddyBearsAmount + 
    minionsAmount + truckToyAmount;

    let puzzlePrice = puzzle * puzzlesAmount;
    let dollPrice = doll * dollsAmount;
    let teddyBearPrice = teddyBear * teddyBearsAmount;
    let minionPrice = minion * minionsAmount;
    let truckPrice = truckToy * truckToyAmount;

    let totalPrice = puzzlePrice + dollPrice + teddyBearPrice + 
    minionPrice + truckPrice;
    
    if (totalAmount> 50 || totalAmount == 50) {
        totalPrice *= 0.75;
    }

    totalPrice *= 0.9;

    if (totalPrice > tripPrice || totalPrice == tripPrice) {
        let remainingMoney = totalPrice - tripPrice;
        console.log(`Yes! ${remainingMoney.toFixed(2)} lv left.`);
    } else {
        let remainingMoney = tripPrice - totalPrice;
        console.log(`Not enough money! ${remainingMoney.toFixed(2)} lv needed.`);
    }
}

toyShop(["40.8","20","25","30","50","10"]);

File: Python/SoftUni PB/Exercises/Python Basics/Conditional Statements - Exercise/Toy_shop.py
excursion_price = float(input())

puzzles_amount = int(input())
speaking_dolls_amount = int(input())
teddy_bear_amount = int(input())
minions_amount = int(input())
toy_truck_amount = int(input())

puzzle_price = 2.6 * puzzles_amount
speaking_doll_price = 3 * speaking_dolls_amount
teddy_bear_price = 4.1 * teddy_bear_amount
minion_price = 8.2 * minions_amount
toy_truck_price = 2 * toy_truck_amount

profit = 0

total_toy_amount = puzzles_amount + speaking_dolls_amount + \
    teddy_bear_amount + minions_amount + toy_truck_amount

toy_price = puzzle_price + speaking_doll_price + teddy_bear_price  \
            + minion_price + toy_truck_price

if total_toy_amount >= 50:
    discount = 0.25 * toy_price
else:
    discount = 0
total_price = toy_price - discount
rent = total_price * 0.1
profit = total_price - rent
remaining_money = abs(profit - excursion_price)
if profit >= excursion_price:
    print(f"Yes! {remaining_money:.2f} lv left.")
else:
    print(f"Not enough money! {remaining_money:.2f} lv needed.")
